tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.ts

```diff
@@ -162,6 +164,7 @@ class TfGraphDashboard extends LegacyElementMixin(PolymerElement) {
             stats="[[_stats]]"
             trace-inputs="[[_traceInputs]]"
             auto-extract-nodes="[[_autoExtractNodes]]"
+            on-color-by-changed="_onBoardColorByChanged"
```

Comment:
Would two way binding on L149 achieve the same thing? Honest question.

Reply:
Yup, done.

I realize I forgot to update tf-graph-app, so updated that, too.